CatAnalyzer/python/leptonSF_cff.py

```diff
@@ -834,5 +834,6 @@ def combineSF(set1, set2, additionalUnc1=0, additionalUnc2=0):
 #2017 SF - ongoing
 muonSFTight94X = combineSF(muonSFTightIdOnly94X, muonSFTightIsoOnly94X)
 electronSFMVAWP8094Xv2 = combineSF( combineSF( electronSFMVAWP80IsoIdOnly94Xv2, electronSFMVAWP80RecoOnly94Xv2 ), electronSFHLTZvtx94X)
+electronSFCutBasedIDTightWP94X = combineSF( combineSF( electronSFCutBasedTightIDOnly94Xv2, electronSFMVAWP80RecoOnly94Xv2), electronSFHLTZvtx94X)
```

Comment:
electronSFCutBasedIDTightWP94X includes Reco, Zvtx SF which is misleading. Change name or remove it.

Reply:
I change name as electronCombinedSFTightWP94X.